### ISSUE TITLE
Feature Request: Error bands for fits

### ISSUE THREAD

**Issue by evanberkowitz** [jeremysanders/veusz#135]
In addition to showing error bars on points, which veusz does extremely well, it would be great to be able to automatically show confidence bands on fits.  For comparison, look under `Applications` at Mathematica's [NonlinearModelFit](http://reference.wolfram.com/language/ref/NonlinearModelFit.html), an example plot shown below.

![](http://reference.wolfram.com/language/ref/Files/NonlinearModelFit.en/O_73.gif)

They don't give an algorithm by which they generate these bands.  It'd be good to be able to configure opacity, color, whether there's a thick line at the boundary (no, it's hideous), confidence level / σ, etc., as well.  This ease of constructing these error bands is the primary reason I still rely on Mathematica for fitting and producing "final" plots.

**Comment by jeremysanders** [comment]
You're right - this would be great to have - I'd like to add sort of more sophisticated fitting code, which would allow the user to do this. It's just a matter of finding time to work on this.

**Comment by evanberkowitz** [comment]
asdf

**Comment by ssgromov** [comment]
+1
